test/release/examples/Makefile

```diff
@@ -26,13 +26,16 @@ clean: FORCE
 benchmarks: FORCE
 	cd benchmarks && $(MAKE)
 
+patterns: FORCE
```

Comment:
I switched to a different system and see the "jobserver unavailable" warnings now - and also that this PR fixes them.

I think this is good to go. Thanks.